chat-plugins/github.js

```diff
@@ -1,30 +1,37 @@
-/*
-	Github plugin by Spandamn
-	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
-	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
-*/
+/**
+ * GitHub Alert Notifications
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This notifies a set of rooms of changes to specific repositor(y/ies).
+ *
+ * Much of this comes from Ecuacion's bot, which is based on xfix's bot,
+ * and jd who originally converted this to PS server-side.
+ *
+ * @license MIT license
+ */
+
 'use strict';
 
-let config = {};
+let gitConfig = {};
 
 if (!Config.github) {
 	return;
 } else if (Config.github === {}) {
-	config.port = 3420;
-	config.secret = "";
+	gitConfig.port = 3420;
+	gitConfig.secret = "";
 } else {
-	config = {
+	gitConfig = {
 		port: Config.github.port,
 		secret: Config.github.secret,
 	};
 }
 
-const git = exports.github = require('githubhook')(config);
+const git = exports.github = require('githubhook')(gitConfig);
 
 let updates = {};
 
 let sendReport = function (html) {
-	if (Config.github && Config.github.rooms) {
+	if (Config.github && Config.github.rooms && Config.github.rooms !== []) {
```

Comment:
`Boolean([])` returns true too, if you want it changed possibly do `&& Config.github.roms.length` since `0` is a falsey value